maths/factorial_iterative.py

```diff
@@ -0,0 +1,19 @@
+# factorial of a number
+
+
+n=int(input("Enter a number"))
+def factorial(n):
```

Comment:
See other functions in this repo to see how type hints and doctests work.  They are required.